{
  "title": "CVE-2020-25635",
  "cve": "CVE-2020-25635",
  "notes": {
    "description": "A flaw was found in Ansible Base when using the aws_ssm connection plugin as garbage collector is not happening after playbook run is completed. Files would remain in the bucket exposing the data. This issue affects directly data confidentiality.",
    "published": "2020-10-05",
    "modified": "2020-10-05"
  },
  "references": [
    {
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25635",
      "description": "CONFIRM:https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25635"
    },
    {
      "url": "https://github.com/ansible-collections/community.aws/issues/222",
      "description": "MISC:https://github.com/ansible-collections/community.aws/issues/222"
    }
  ]
}
